andela/ah-kifaru-backend
# #166840916 Implement user email verification notification on successful signup

#### What does this PR do?
- add the feature for user to receive a verification link via email upon successful registration
#### Description of Task to be completed?
  -  SendGrid account
  -  Setup email template on SendGrid
  -  Implement mailer function to send mail

#### How should this be manually tested?
 - Clone the project, run `npm install` to install all dependencies and  run `npm start`
 - When a user signs up for an account using `/api/v1/auth/signup` endpoints they get an email address telling them to verify their account through the click of a button.
#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#166840916](https://www.pivotaltracker.com/story/show/166840916)
#### Screenshots (if appropriate)
